avocado/core/loader.py

```diff
@@ -386,6 +387,10 @@ def load_test(self, test_factory):
         else:
             test_path = None
         if isinstance(test_class, str):
+            # instrumented tests (only) accept the setup_attempts parameter
+            setup_attempts = settings.get_value("runner", "instrumented", int,
```

Comment:
Hmm... check this statement. I think it should start with:

settings.get_value("runner.instrumented", "setup_attempts", key_type=int, ....

Reply:
True! I probably didn't catch that because the hardcoded defaults match the configurable one.  Thanks!